deploy_and_run_integration_tests.sh

```diff
@@ -25,17 +25,21 @@ set -euo pipefail
 #
 # - The user should have access to the 'gcp-variant-transforms-test' project.
 
-PROJECT="gcp-variant-transforms-test"
-GS_DIR="integration_test_runs"
 GREEN="\e[32m"
 RED="\e[31m"
 TEST_ARGUMENTS=""
+GS_DIR_OPT="--gs_dir"
 KEEP_IMAGE_OPT="--keep_image"
 IMAGE_TAG_OPT="--image_tag"
+PROJECT_OPT="--project"
+RUN_UNIT_TEST_OPT="--run_unit_tests"
 SKIP_BUILD_OPT="--skip_build"
 image_tag=""
 keep_image=""
 skip_build=""
+gs_dir="integration_test_runs"  # default GS dir to store logs, etc.
```

Comment:
should update usage() too?

Reply:
Done.
Heh, I actually updated usage last night, after sending the PR, but forgot to push it to GitHub, I guess I was too tired :-)